<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>productdetails</class>
 <widget class="QDialog" name="productdetails">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>498</width>
    <height>718</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>472</width>
        <height>692</height>
       </rect>
      </property>
      <widget class="QLabel" name="label_menubar">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>481</width>
         <height>71</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">
background-color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QTextBrowser" name="textBrowser">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>20</y>
         <width>221</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Al Nile</family>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(0, 0, 0);</string>
       </property>
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Al Nile'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'.AppleSystemUIFont'; font-size:24pt; font-weight:700; color:#63d641;&quot;&gt;ECO WARDROBE&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>0</y>
         <width>71</width>
         <height>71</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>../Downloads/EcoWardrobe-01.jpg</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QGroupBox" name="groupBox">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>80</y>
         <width>301</width>
         <height>201</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Gotu</family>
         <pointsize>17</pointsize>
         <italic>true</italic>
         <bold>true</bold>
         <underline>false</underline>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">
background-color: rgb(188, 255, 158)</string>
       </property>
       <property name="title">
        <string>Plaid Shirt (Black and White)</string>
       </property>
       <widget class="QTextBrowser" name="textBrowser_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>281</width>
          <height>161</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Annai MN</family>
         </font>
        </property>
        <property name="html">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Annai MN'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'.AppleSystemUIFont';&quot;&gt;Embrace eco-conscious fashion without compromising on style with our Eco-Friendly Plaid Overshirt. Crafted with the utmost respect for the environment, this on-trend overshirt embodies a commitment to sustainability. Each garment is uniquely constructed from high-quality recycled textiles sourced from BMCC's innovative recycling program.&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'.AppleSystemUIFont';&quot;&gt;&lt;br /&gt;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>141</width>
         <height>231</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>../Downloads/001e6bebf2df713d705b25bd3270c71d.jpg</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_priceAmount">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>290</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>$18</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_minus">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>330</y>
         <width>21</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>-</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_plus">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>330</y>
         <width>21</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>+</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_AddToCart">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>310</y>
         <width>121</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>Add to cart</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>310</y>
         <width>51</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>../Downloads/9eb31f1397aa013972c607f8d38923fd.jpg</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_6">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>620</y>
         <width>101</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(188, 255, 158)</string>
       </property>
       <property name="text">
        <string>Back</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_7">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>620</y>
         <width>111</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">
background-color: rgb(188, 255, 158)</string>
       </property>
       <property name="text">
        <string>Purchase</string>
       </property>
      </widget>
      <widget class="QLabel" name="Label_number">
       <property name="geometry">
        <rect>
         <x>410</x>
         <y>20</y>
         <width>41</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>../Downloads/04381104d75831ebbf960402d4f6dd5b.jpg</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>370</y>
         <width>451</width>
         <height>101</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Kavivanar</family>
         <pointsize>12</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">
background-color: rgb(250, 255, 203)</string>
       </property>
       <property name="text">
        <string>Materials:

100% recycled cotton blend
Upcycled yarns for durable stitching
Biodegradable buttons made from recycled mateials</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>480</y>
         <width>451</width>
         <height>121</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Kavivanar</family>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">
background-color: rgb(250, 255, 203)</string>
       </property>
       <property name="text">
        <string>Sustainability Impact:

Reduced water consumption by using recycled materials
Lowered carbon footprint through local sourcing and production
Promotion of circular fashion by repurposing BMCC's recycled textiles
Contribution to waste reduction within the fashion industry</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_number">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>330</y>
         <width>21</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>310</y>
         <width>58</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Free Size</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_home">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>30</y>
         <width>51</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Home</string>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
